### DISCUSSION TITLE
Variants API Support

### DISCUSSION THREAD

**Discussion by artmsilva** [facebook/stylex#143] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**

We are considering StyleX for our design system. We are moving away from Stitches.js. However there are a lot of features of Stitches.js we like such as their "variants" API and "$<token" syntax.

**Describe a solution you'd like**
It's not really a problem or a feature request. Curious if the maintainers know if it's possible with StyleX to build something similar either internally or externally atop of StyleX's API.

**Describe alternatives you've considered**

**Additional context**

**Comment by necolas** [comment, marked as answer]
The pattern for variants is documented here https://stylexjs.com/docs/learn/styling-ui/using-styles/#style-variants
